src/sentry/db/models/base.py

```diff
@@ -267,6 +267,10 @@ def write_relocation_import(
         self.save(force_insert=True)
         return (self.pk, ImportKind.Inserted)
 
+    @classmethod
+    def query_manager_to_use_for_deletion(cls):
```

Comment:
Added this because when we delete tasks through ScheduledDeletion, and there exists a manager that filters out certain values for API/UI purposes, the object to delete can get filtered out. This lets us add a special query manager for when we use scheduled deletions